### ISSUE TITLE
Create non-existent folders when saving images

### ISSUE THREAD

**Issue by jordanbtucker** [comfyanonymous/ComfyUI#64]
When I set the filename prefix to `testing/dog` or `testing\dog`, no images are saved to the output folder. Instead, an error is logged in the console.

```
FileNotFoundError: [Errno 2] No such file or directory: '...\\ComfyUI\\output\\testing\\dog_00001_.png'
```

Creating a `testing` folder in `output` fixes the error. It would be great if ComfyUI could create any missing folders for the user instead.

**Comment by jordanbtucker** [comment]
Looks like there is a PR for this already.

Fixed by #60

**Comment by jordanbtucker** [comment]
I just realized that creating the folder manually does not properly index the files when they are saved. All files start with the index `1`, so they are overwritten each time. #60 fixes this issue too.

**Comment by comfyanonymous** [comment]
This is solved by #60
